torch/distributed/optim/__init__.py

```diff
@@ -7,4 +7,6 @@
 """
 from .optimizer import DistributedOptimizer
 from .post_localSGD_optimizer import PostLocalSGDOptimizer
-from .zero_redundancy_optimizer import ZeroRedundancyOptimizer
+# For backwards compatibility, please import from torch.distributed.zero
```

Comment:
nit: On initial read, I interpreted this as something like "In order to preserve [For] backwards compatibility, please import from ... instead" rather than something like "the following is for backwards compatibility, please import from ..."
Maybe say:
```
# Included for backwards compatibility, please import from torch.distributed.zero
# instead.
```
Up to you.